crates/edr_eth/src/log/receipt.rs

```diff
@@ -13,6 +13,7 @@ pub struct ReceiptLog {
     #[cfg_attr(feature = "serde", serde(flatten))]
     pub inner: Log,
     /// transaction hash
+    // TODO review: this is optional in Hardhat
```

Comment:
Same as the other one. We encoded optionality in static types: `Log` vs `ReceiptLog`